LinhNT/Network/Switch/bonding.md

```diff
@@ -0,0 +1,28 @@
+## Bonding
+___
+> **Bonding** là công nghệ giúp tăng BW + reduntdancy bằng cách kết hợp 2 hay nhiều network interfaces thành **1 logical interface**. 
+
+### Kịch bản
+Ta tạo một kịch bản gồm 4 hosts như theo hình dưới đây.Còn h2 như một switch.  Chúng ra sẽ **bond** 2 interfaces của h1 thành 1 logical interface. Chúng ta sẽ đánh giá hiệu năng của 2 phần, khi có **bonding** và khi **không bonding**.
+
+##### Kịch bản 1: Không thực hiện bonding
```

Comment:
Em chưa nói rõ thực hiện test băng thông thế nào, đầu nào là server, đầu nào là client. Với chưa có kịch bản down 1 đường sẽ thế nào, để chuyển được toàn bộ băng thông sang 1 đường thì mất bao lâu, bao nhiêu packet bị loss,...